README.md

```diff
@@ -1 +1,139 @@
-# Stock-Metrics-WebApp
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Stock-Metrics-WebApp</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] As required for the project, kindly ensure that you deploy the application, you can use `Render`, `Netlify` or `GitHub Pages`,....  Your deployed project should work properly, the home and detail page works as expected and the link to the deployed project should present in the README in the appropriate section(`[🚀 Live Demo](#live-demo)`).